streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -496,10 +493,6 @@ public class StreamsConfig extends AbstractConfig {
     public static final String BUFFERED_RECORDS_PER_PARTITION_CONFIG = "buffered.records.per.partition";
     public static final String BUFFERED_RECORDS_PER_PARTITION_DOC = "Maximum number of records to buffer per partition.";
 
-    /** {@code built.in.metrics.version} */
-    public static final String BUILT_IN_METRICS_VERSION_CONFIG = "built.in.metrics.version";
-    private static final String BUILT_IN_METRICS_VERSION_DOC = "Version of the built-in metrics to use.";
```

Comment:
We cannot make any changes to `StreamsConfig` as it would be a public API change (which requires a KIP).

As pointed out on the Jira ticket, we can just keep it, to avoid the need to do another KIP to re-introduce this parameter if we would need it again.